src/components/ContributorCard.tsx

```diff
@@ -34,12 +34,18 @@ export default function ContributorCard({ data }: { data: Card }) {
         className="rounded-full border"
       />
       <div className="mb-2 flex gap-12 text-2xl">
-        <Link href={data.html_url} target="_blank" className="duration-150 hover:text-[#411AFF]">
+        <Link href={data.html_url} target="_blank"                   
+                  aria-label="Visit us on GitHub"
```

Comment:
```suggestion
                  aria-label="GitHub Profile"
```

This is personal GitHub profile, so this would be better.